services/QuillLMS/app/assets/stylesheets/pages/progress_reports/diagnostic_student_responses_index.scss

```diff
@@ -15,7 +15,7 @@
       th {
         height: 100%;
         &:not(:last-child) {
-          border-right: 1px solid #CFCFCF;
+          border-right: 1px solid #CFCFCF !important;
```

Comment:
Should this color be a variable if we have one for it?

Reply:
yes, but this actually shouldn't be in this PR-- removing this file now